src/components/results.js

```diff
@@ -8,16 +8,34 @@ export default class Results extends Component {
 		super(props)
 		this.state = {
 			data: {},
-			canVote: 'Yes',
+			canVote: null,
 		};
 	}
 
 	componentDidMount(props) {
 		console.log(this.props)
 		console.log('name: ' + this.props.name);
-		this.getStates()
+		this.yesNoMaybe();
 	}
 
+
+     //Method to determine if you can vote
+
+    yesNoMaybe() {
+            console.log("yesNoMaybe")
+            if (this.props.age >= 18 && this.props.crimes === 'No' && this.props.citizen === 'Yes') {
+                console.log("yesNoMaybe.yes")
```

Comment:
Please remove all of the `console.log` calls. Those are great to use for developing and testing, but as we move the app towards production, we'll want to remove them all.